### DISCUSSION TITLE
Remote TCPIP options

### DISCUSSION THREAD

**Discussion by Naga3028-oss** [sanny32/OpenModSim#22] [Q&A] [answered]
Hi Team, 

 I have a doubt, is there any way we can use Remote TCPIP, something like instead of simulating a tag for local host i want to make Open modsim to be simulator for different ip,is it possible anyhow.
![MODBUS](https://github.com/user-attachments/assets/b6c85a15-e05f-4a0f-b506-c84f80cd3fe0)
![MODBUS1](https://github.com/user-attachments/assets/d72892cd-2204-4167-b41e-b7f39297b231)

**Comment by sanny32** [comment]
Hello @Naga3028-oss . The Open Modsim program accept connections from any interface. There is no way to change this now.
But you can change this in sources. See [https://github.com/sanny32/OpenModSim/blob/main/omodsim/connectiondetails.h](https://github.com/sanny32/OpenModSim/blob/main/omodsim/connectiondetails.h) `TcpConnectionParams` struct `const QString IPAddress = "0.0.0.0"` line. Change `0.0.0.0` to your IP Address and rebuild program.

**Reply by Naga3028-oss** [reply]
Hello Sanny32,
Thanks for instant reply.
 The Link you provided seems to be broken, can you please verify.

**Reply by sanny32** [reply]
Link was corrected.

**Reply by Naga3028-oss** [reply]
Hi Sanny,

 Thanks it works now.

**Comment by sanny32** [comment]
This functionality will be available in version 1.7

**Reply by Naga3028-oss** [reply]
Hi Sanny,  

 When can we expect the release of version 1.7, as i am purely Non IT guy, its very confusing like the term you used "rebuild the program", so hoping in Ver 1.7 it should be easily customizable.

**Reply by sanny32** [reply, marked as answer]
Hello @Naga3028-oss. You can download version 1.7 from https://github.com/sanny32/OpenModSim/releases/tag/1.7.0

**Reply by Naga3028-oss** [reply]
Hi Sanny ,

 Thanks man your the real life saver.
